src/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const DPRINT_CONFIG_FILEPATH_GLOB = "**/{dprint,.dprint}.{json,jsonc}";
+export const DPRINT_CONFIG_FILEPATH_GLOB = "**/{dprint,.dprint,dprint.*,.dprint.*}.{json,jsonc}";
```

Comment:
Does the extension work with multiple configs in the same folder? Why not use dprint.config? I don't want to support non-standard config paths by default like this.